app/config/settings_prod.py

```diff
@@ -216,6 +216,8 @@
     # permissions to change ACL, then this setting must be None
     AWS_DEFAULT_ACL = None
     AWS_S3_SIGNATURE_VERSION = "s3v4"
+    # The path for the pingdom healthcheck
+    HEALTHCHECK_ENDPOINT = os.environ.get('HEALTHCHECK_ENDPOINT', None)
```

Comment:
This should not be part of the AWS settings but outside this try except block. Then I would raise an error with a correct message if no HEALTHCHECK_ENDPOINT is set.